src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -384,7 +386,9 @@ public static void writeBundle(OutputStream outputStream, final List<? extends C
                 // Generate the content of the manifest.md going trough all the components which will be included. It
                 // also returns the contents to include. We pass maybeFilter to filter the names written in the manifest
                 appendManifestHeader(manifest);
+                long startTime = System.currentTimeMillis();
                 List<Content> contents = appendManifestContents(manifest, errorWriter, components, componentConsumer, maybeFilter);
+                LOGGER.log(Level.FINE, "Took " + (System.currentTimeMillis()-startTime) + "ms to process all components");
```

Comment:
Or https://javadoc.jenkins.io/hudson/Util.html#getTimeSpanString(long) in case this ever reaches into the seconds!

Reply:
I'd actually like to keep in ms in the logs, following what we do in core startup performance logs. Keeping a single unit helps in troubleshooting to exploit the logs (do a cat | grep | sed | sort ..)

Not that running this in some env, the processing or components may take some time. Component adding `FileContent`s for example:

```
2023-05-09 06:18:37.906+0000 [id=329]	FINE	c.c.j.support.SupportPlugin#appendManifestContents: Took 245ms to process component Jenkins Global Configuration File (Encrypted secrets are redacted)
[...]
2023-05-09 06:18:40.596+0000 [id=329]	FINE	c.c.j.support.SupportPlugin#appendManifestContents: Took 2690ms to process component Other Jenkins Configuration Files (Encrypted secrets are redacted)
```